### PR TITLE
Change amp-youtube embed to consistently use data-attributes for conf…

### PR DESCRIPTION
**Pull request by sriramkrish85**
…iguration.

### CODE REVIEW

File: extensions/amp-youtube/0.1/amp-youtube.js [hunk 1]
```diff
@@ -34,8 +34,10 @@ class AmpYoutube extends AMP.BaseElement {
   layoutCallback() {
     var width = this.element.getAttribute('width');
     var height = this.element.getAttribute('height');
-    var videoId = AMP.assert(this.element.getAttribute('video-id'),
-        'The video-id attribute is required for <amp-youtube> %s',
+    var videoId = AMP.assert(
+        (this.element.getAttribute('data-videoid') ||
+        this.element.getAttribute('video-id')),
```
Comment (erwinmombay) [inline extensions/amp-youtube/0.1/amp-youtube.js]: shouldnt it be `videoid` for consistency?

Comment (cramforce) [inline extensions/amp-youtube/0.1/amp-youtube.js]: This is only for backward compat. Lets add a comment to that respect.

File: examples/everything.amp.html [hunk 2]
```diff
@@ -151,11 +151,11 @@ <h1 id="top">AMP #0</h1>
 
   <h2>Youtube</h2>
   <amp-youtube
-      video-id="mGENRKrdoGY"
+      data-videoid="mGENRKrdoGY"
       width="480" height="270"></amp-youtube>
 
   <amp-youtube
-      video-id="mGENRKrdoGY"
+      data-videoid-videoid="mGENRKrdoGY"
```
Comment (cramforce) [inline examples/everything.amp.html]: Double :)

File: extensions/amp-youtube/amp-youtube.md [hunk 3]
```diff
@@ -31,7 +31,8 @@ With responsive layout the width and height from the example should yield correc
 
 #### Attributes
 
-**video-id**
+**data-videoid**
+~~video-id~~
```
Comment (cramforce) [inline extensions/amp-youtube/amp-youtube.md]: Lets just remove it.

**Comment by sriramkrish85** [comment]
PTAL

**Comment by cramforce** [comment]
LGTM

**Pull request merged**